### PR TITLE
GSYE-724: Configure hours of delay that was previously a constant

### PR DESCRIPTION
**Pull request by BigTava**
## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/gsy_e_core/rq_job_handler.py [hunk 1]
```diff
@@ -160,6 +160,8 @@ def _configure_constants_constsettings(
     ConstSettings.SCMSettings.MARKET_ALGORITHM = CoefficientAlgorithm(
         settings.get("scm_coefficient_algorithm", 1)).value
 
+    ConstSettings.SCMSettings.HOURS_OF_DELAY = settings.get("scm_cn_hours_of_delay", 72)
```
Comment (hannesdiedrich) [inline src/gsy_e/gsy_e_core/rq_job_handler.py:163]: Why is this needed here?

**Review by hannesdiedrich — COMMENTED** [review]
One question, once addressed, LGTM

File: src/gsy_e/gsy_e_core/rq_job_handler.py (hunk 1 above)
Comment (spyrostz) [reply src/gsy_e/gsy_e_core/rq_job_handler.py:163]: Same as Hannes, I also believe that this is no longer required. Probably only useful if we want to somehow access this variable globally, however I do not foresee the need for it.

**Review by spyrostz — COMMENTED** [review]
Same comment as Hannes, LGTM once resolved.

File: src/gsy_e/gsy_e_core/rq_job_handler.py (hunk 1 above)
Comment (BigTava) [reply src/gsy_e/gsy_e_core/rq_job_handler.py:163]: Exactly. Thanks

**Review by spyrostz — APPROVED** [review]
LGTM
